Products/PortalTransforms/setuphandlers.py

```diff
@@ -27,23 +27,23 @@ def correctMapping(out, portal):
                         print >> out, "...ok"
 
 
-def updateSafeHtml(out, portal):
-    print >> out, 'Update safe_html...'
-    safe_html_id = 'safe_html'
-    safe_html_module = "Products.PortalTransforms.transforms.safe_html"
+def updateTransform(out, portal, id):
+    print >> out, 'Update {0}...'.format(id)
```

Comment:
I am pretty sure this does not work any longer in Python 3 (I know, this is used all over the place at the moment, but we plan to go Python 3 soonish). I'd propose to use: 
``out.write('Update {0}...\n'.format(transform_id))``